test-suite/output/notation_principal_scope.out

```diff
@@ -0,0 +1,20 @@
+The command has indeed failed with message:
+Argument X was previously inferred to be in scope function_scope but is here
+used in the empty scope stack. Scope function_scope will be used at parsing
+time unless you override it by annotating the argument with an explicit scope
+of choice. [inconsistent-scopes,syntax]
```

Comment:
This warning makes me realize that there is no syntax for choosing the "empty scope stack" alternative. So, maybe, the "in scope foo" should also include a variant "in no scope"??

Reply:
Not mandatory, but just in case the comment was not seen, I believe it would complete the picture to support a "no scope" modifier.